src/sidechaindb.h

```diff
@@ -2,8 +2,8 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#ifndef BITCOIN_SIDECHAINDB_H
```

Comment:
These statements at the top of headers in C++ (called include guards) are supposed to be the filename, so this actually should stay as BITCOIN_SIDECHAINDB_H. It's a style decision how these are actually named, but to stay in line with Bitcoin these should remain BITCOIN_SIDECHAINDB_H.